home-server-generator/pom.xml

```diff
@@ -86,6 +86,7 @@
                                 <parentArtifactId>${project.parent.artifactId}</parentArtifactId>
                                 <parentGroupId>${project.parent.groupId}</parentGroupId>
                                 <parentVersion>${project.parent.version}</parentVersion>
+                                <useBeanValidation>false</useBeanValidation>
```

Comment:
I don't think we need to disable it but configure properly. Also, I don't think it should be included in these efforts.